test/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Container for running Couchbase benchmarks
+FROM node:slim
```

Comment:
Is it important that this Dockerfile live in this repo? My tendency would be to put it in a separate repo.